README.md

```diff
@@ -34,21 +34,56 @@ Refer to these files by adding a `<link>` and a `<script>` element into your HTM
 
 ### Install with NPM
 
-If you have `node` installed on your machine, you can use `npm` to install the Web Design Standards. In your `package.json` list the following under `dependencies`:
+If you have `node` installed on your machine, you can use `npm` to install the
+Draft Web Design Standards.
 
-```json
-  "uswds": "git@github.com:18F/web-design-standards.git#v0.9.0"
 ```
+cd path/to/project-using-npm
+npm install --save uswds
+```
+
+This will add the Draft Web Design Standards as a dependency for your project and save
+it to your `package.json` file. The package will be installed in `node_modules`
+under `uswds`.
+
+Refer to these files by adding a `<link>` and a `<script>` element into your HTML pages:
 
-On subsequent runs of `npm install`, the package will be installed in `node_modules` under `uswds`. The CSS will be *not* compiled after it is installed. Instead, you are able to compile it however it fits into your project.
+```
+<link rel="stylesheet" href="./node_modules/uswds/assets/css/uswds.css">
+<link rel="stylesheet" href="./node_modules/uswds/assets/css/google-fonts.css">
```

Comment:
Doesn't this file include 8 fonts? When we discussed fonts originally we decided to limit the default to 3 or 4 or 5 (can't actually remember what we agreed on) to limit performance problems. Should we not be recommending people to include all 8 fonts if they don't need them.

On my cloud.gov project where I imported USWDS, I ended up creating my own fonts file with just 3 `@font-face` links included.